reactor-core/src/main/java/reactor/core/publisher/Traces.java

```diff
@@ -103,7 +102,7 @@ static String extractOperatorAssemblyInformation(String source) {
 	}
 
 	static boolean isUserCode(String line) {
-		return !line.startsWith("reactor.core.publisher") || line.contains("Test");
+		return !line.startsWith(PUBLISHER_PACKAGE_PREFIX) || line.contains("Test");
```

Comment:
In case at any point future maintainers decide to add a package that begins with reactor.core.publisher but does not end with a dot I think we can remove the dot from the prefix. Unless it's needed?

Reply:
No, that's ok, let's leave it as is.